crates/burn-import/onnx-tests/tests/test_onnx.rs

```diff
@@ -2183,6 +2187,46 @@ mod tests {
         assert_eq!(expected_shape, output.shape());
     }
 
+    #[test]
+    fn add_constant_f32() {
+        let device = Default::default();
+        let model = constant_f32::Model::<Backend>::new(&device);
+        let input = TensorData::zeros::<f32, _>(Shape::from([2, 3, 4]));
+        let expected_output = TensorData::full::<f32, _>(Shape::from([2, 3, 4]), 2f32);
+        let output = model.forward(input.into());
+        assert_eq!(expected_output, output.to_data());
+    }
+
+    #[test]
+    fn add_constant_f64() {
+        let device = Default::default();
+        let model = constant_f64::Model::<Backend>::new(&device);
+        let input = TensorData::zeros::<f64, _>(Shape::from([2, 3, 4]));
+        let expected_output = TensorData::full(Shape::from([2, 3, 4]), 2f32);
```

Comment:
I'm not sure if the addition in the model is coercing f64 -> f32 (and i32 -> i64 below). I wasn't sure how to get PyTorch to just forward the constant so these tests use the add operation also